services/lightspark_client.go

```diff
@@ -242,6 +242,24 @@ func (client *LightsparkClient) CreateNodeWalletAddress(nodeId string) (string,
 	return walletAddress, nil
 }
 
+func (client *LightsparkClient) CreateNodeWalletAddressWithKeys(nodeId string) (*objects.CreateNodeWalletAddressOutput, error) {
```

Comment:
I need to make a new function for this as it would be a breaking change for updating the existing one.